src/db/options.rs

```diff
@@ -112,8 +111,9 @@ pub enum ValidationAction {
 }
 
 /// Specifies default configuration for indexes created on a collection, including the _id index.
-#[derive(Clone, Debug, PartialEq, Serialize)]
+#[derive(Clone, Debug, TypedBuilder, PartialEq, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
+#[non_exhaustive]
 pub struct IndexOptionDefaults {
```

Comment:
This struct was missing a few attributes so I added them. I don't think any of our users will use this though.

Reply:
TIL `create` accepts `indexOptionDefaults`